finat/aw.py

```diff
@@ -27,23 +29,39 @@ def _edge_transform(T, coordinate_mapping):
     J_np = numpy.array([[J[0, 0], J[0, 1]],
```

Comment:
Is there syntactic sugar in gem to make this cleaner (so you don't have to explicitly index/wrap)?